package.json

```diff
@@ -0,0 +1,63 @@
+{
```

Comment:
Can we add an icon please? Doesn't look great without one on the VS Marketplace. Could just be the styled Q company logo. Similarly could we add a matching banner?